src/main/java/com/lorepo/icplayer/client/ui/PlayerView.java

```diff
@@ -12,8 +12,10 @@
 import com.google.gwt.user.client.ui.HorizontalPanel;
 import com.google.gwt.user.client.ui.PopupPanel;
 import com.google.gwt.user.client.ui.VerticalPanel;
+import com.lorepo.icf.utils.JavaScriptUtils;
```

Comment:
jeżeli ticket znowu wróci, to pamiętaj żeby to wyrzucić.

Reply:
Ponawiam prośbę, jeżeli wróci